chief-data-officers.md

```diff
@@ -15,7 +15,7 @@ The Chief Data Officer and Chief Data Scientist are an emerging roles in US Fede
 {: .table .table-striped}
 CFO Act Agencies                                |  Agency / Bureau                                |  Chief Data Officer             
 --------------                                  | --------------                                  | --------------                  
-Consumer Financial Protection Agency            |                                                 | Linda Powell
+Consumer Financial Protection Bureau            |                                                 | Linda Powell
```

Comment:
Thanks, @chrisMurphyTCG!

While we're here, Linda Powell has departed the Bureau. The acting CDO is Andy Thatai.

```suggestion
Consumer Financial Protection Bureau            |                                                 | Andy Thatai (acting)
```